python/tests/test_server.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from remotebmi.client.client import RemoteBmiClient
+from remotebmi.reserve import (
+    reserve_grid_nodes,
+    reserve_grid_shape,
+)
+from remotebmi.server import make_app
+
+from .fake_models import Rect3DGridModel
```

Comment:
Maybe dummy models would be a nicer name than fake. As you do test the actual bmi responses.
But it's fine to leave it as is.

Reply:
I am using fake from definition at https://martinfowler.com/articles/mocksArentStubs.html . more describing